packages/react-native/Libraries/Core/setUpTimers.js

```diff
@@ -10,6 +10,8 @@
 
 'use strict';
 
+import NativeIdleCallbacks from '../../src/private/webapis/idlecallbacks/specs/NativeIdleCallbacks';
```

Comment:
We don't automatically enable inline requires in files within `react-native`, so we should move this to be a `require` call within the polyfill to avoid initializing the native module unless the API is actually used.